sql-database/failover-groups/add-single-db-to-failover-group-az-cli.sh

```diff
@@ -0,0 +1,112 @@
+﻿#!/bin/bash
+# Set variables
+export subscriptionID=<SubscriptionID>
+export resourceGroupName=myResourceGroup-$RANDOM
+export location=WestUS2
+export adminLogin=azureuser
+export password=`openssl rand -base64 16`
+export serverName=mysqlserver-$RANDOM
+export databaseName=mySampleDatabase
+export drLocation=EastUS2
+export drServerName=mysqlsecondary-$RANDOM
+export failoverGroupName=failovergrouptutorial-$RANDOM
+
+# The ip address range that you want to allow access to your DB. 
+# Leaving at 0.0.0.0 will prevent outside-of-azure connections
+export startip=0.0.0.0
+export endip=0.0.0.0
+
+# Print out randomized variables
```

Comment:
When displaying randomized values, make it clear to the user which values correspond to what elements are being created.

Reply:
Sorry, would you mind clarifying what the expectation is here?

Reply:
As part of the `echo`, tell the user what the value is used for.